deploy/cephcsi/image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ceph/ceph:v14.2
+FROM ceph/ceph:v15.2
```

Comment:
Why stick to .2 and not use the latest stable v15?

Reply:
done updated to v15